### PR TITLE
Add RNA-SeQC to Quantification Section

### PR DESCRIPTION
**Pull request by dandyrilla**
I've added additional quantification software.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ The following items allow for scalable genomic analysis by introducing specializ
 
 - **[Cufflinks](https://github.com/cole-trapnell-lab/cufflinks)** - Cufflinks assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples. [ [paper-2010](https://www.nature.com/articles/nbt.1621) ]
 - **[RSEM](https://github.com/deweylab/RSEM)** - A software package for estimating gene and isoform expression levels from RNA-Seq data. [ [paper-2011](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-323) | [web](http://deweylab.github.io/RSEM/) ]
+- **[RNA-SeQC](https://software.broadinstitute.org/cancer/cga/rna-seqc)** - RNA-seq metrics for quality control and process optimization. [ [paper-2012](https://academic.oup.com/bioinformatics/article/28/11/1530/267467) | [web](https://software.broadinstitute.org/cancer/cga/rnaseqc_download) ]
```
Comment (TGotwig) [inline README.md:150]: Hello dandyrilla 👋
```suggestion
 - **[RNA-SeQC](https://github.com/getzlab/rnaseqc)** - RNA-seq metrics for quality control and process optimization. [ [paper-2012](https://academic.oup.com/bioinformatics/article/28/11/1530/267467) | [web](https://software.broadinstitute.org/cancer/cga) ]
```

**Review by TGotwig — CHANGES_REQUESTED** [review]

**Comment by dandyrilla** [comment]
@TGotwig Thanks for suggesting!

**Comment by TGotwig** [comment]
@dandyrilla Please also create `.markdownlint.json` with the following content 😊

```json
{
  "line-length": false
}
```

**Comment by dandyrilla** [comment]
> @dandyrilla Please also create `.markdownlint.json` with the following content 😊
> 
> ```json
> {
>   "line-length": false
> }
> ```

Did I contain any line causing markdown syntax error?
I thought that the line causing error is not mine.
@TGotwig , Could you help me how to add `.markdownlint.json` to fix it up?

**Comment by TGotwig** [comment]
@dandyrilla It's just a file which should be created inside the root folder of this project.

**Pull request closed without merging**